rpm/sailfish-setup.spec

```diff
@@ -73,6 +73,8 @@ if ! getent passwd sailfish-fingerprint >/dev/null ; then
     useradd -r -g sailfish-authentication -d / -s /sbin/nologin sailfish-fingerprint || :
 fi
 
+useradd -U qr || :
```

Comment:
I'm not sure if we still even need this group but in any case, we should not have custom groups named like this. Instead it should be sailfish-qr for example.